### DISCUSSION TITLE
IP Rules prioritization

### DISCUSSION THREAD

**Discussion by jazer143** [postalserver/postal#2011] [Help with using Postal]
Hi, Good day! May I ask anyone here your expertise/experience about how Postal prioritizes IP rules.

What takes precedence:
1. Organization IP rule vs SMTP server IP rule
2. To addresses and from address (in Org or server) e.g.
        a.  IP rule 1: FROM address is aaa.com, IP pool A
        b. IP rule 2: TO address is bbb.com, IP pool B 

Thank you!

regards,
Jazer

**Comment by willpower232** [comment]
I would imagine that the server IP Pool takes precedence over the organization IP Pool but if you're doing both then I'd be worried about causing some confusion within Postal.

I'm not sure whether you can configure specific to/from addresses within the rules, WYSIWYG though.

**Reply by jazer143** [reply]
Hi @willpower232 ,

I have an IP rule which is the FROM address is sending to any domains, but there are some ESP e.g. mail.ru that blocking our IPs even if it's not blacklisted at all. I don't know why they do that. So I just want to divert mail.ru to another IP address but this is another IP rule which is in TO address. It's not sitting in one rule. I don't know if Postal reads the top rule first then so on and so forth. 

Thank you by the way.
